lib/auth/webauthn/attestation.go

```diff
@@ -78,11 +91,28 @@ func verifyAttestation(cfg *types.Webauthn, obj protocol.AttestationObject) erro
 	// so both checks (allowed and denied) may be true for the same cert.
 	allowed := len(cfg.AttestationAllowedCAs) == 0
 	for _, cert := range attestationChain {
-		if _, err := cert.Verify(x509.VerifyOptions{Roots: allowedPool}); err == nil {
+		opts := verifyOptsBase // take copy
+		opts.Roots = allowedPool
+		if _, err := cert.Verify(opts); err == nil {
 			allowed = true // OK, but keep checking
+		} else {
+			log.DebugContext(context.Background(),
```

Comment:
Added some logging to help future debugging. The first certificate in the Hello chain fails validation, which is good to know (the second passes so thankfully it works).

For the curious, the error is "x509: unhandled critical extension".